ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -1152,35 +1152,35 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: osd-ccs-aws
-- as: aws-rosa-sts-hypershift-mgmt-p2-f2
-  cron: 28 17 2,4,5,8,10,12,14,16,18,20,22,24,26,28,30 * *
+- as: aws-rosa-sts-hypershift-guest-p1-f7
+  cron: 53 8 1,3,5,7,9,11,13,14,17,19,21,23,25,27,29 * *
```

Comment:
This does not looks like a correct cron entry for f7, is it ?

Reply:
Good catch! Thanks a lot!